testdata/resolvers/tasks/resolver-test-task.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: resolver-test-task
+spec:
+  steps:
+    - name: echo
+      image: alpine
```

Comment:
use `image-registry.openshift-image-registry.svc:5000/openshift/golang` instead of `alpine`